src/app/tip-sheets/content/tip-sheet-1.html

```diff
@@ -22,16 +22,12 @@ <h2 translate>Set aside time to spend with each child</h2>
         the same time eachday so children or teenagers can look forward to it.
       </p>
     </div>
-    <button
-      translate
-      [routerLink]="['/activities']"
-      style="
+    <button translate [routerLink]="['/activities']" style="
```

Comment:
@michaelclapham should we have the translate directives on the router links? I thought this is a path to where activities are and not the text on the button